docker-compose.yml

```diff
@@ -1,47 +1,73 @@
 # DO NOT USE IN PRODUCTION. THIS FILE IS ONLY FOR TESTING PURPOSES.
-version: "2"
+version: "3"
 services:
-    kandalf:
-        image: quay.io/hellofresh/kandalf
-        volumes:
-            - "./_build/out/linux:/app"
-            - "./_build/resources:/config"
-        environment:
-            LOG_LEVEL: "debug"
-            RABBIT_DSN: "kandalf-rabbit:5672"
-            STORAGE_DSN: "redis://kandalf-redis:6379/?key=kandalf"
-            KAFKA_BROKERS: "kandalf-kafka:9092"
-        links:
-            - kafka
-            - redis
-            - rmq
-        networks:
-            - kandalf
+  kandalf:
+    image: quay.io/hellofresh/kandalf
+    volumes:
+      - "./_build/out/linux:/app"
+      - "./_build/resources:/config"
+    environment:
+      LOG_LEVEL: "debug"
+      RABBIT_DSN: "rabbit:5672"
+      STORAGE_DSN: "redis://redis:6379/?key=kandalf"
+      KAFKA_BROKERS: "kafka:29092"
+    depends_on:
+      - kafka
+      - redis
+      - rabbit
 
-    kandalf-kafka:
-        image: spotify/kafka
-        ports:
-            - 9092
-        environment:
-            TOPICS: "new-orders,loyalty"
-        networks:
-            - kandalf
+  zookeeper:
+    image: confluentinc/cp-zookeeper:latest
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
 
-    kandalf-redis:
-        image: redis:3.2-alpine
-        ports:
-            - 6379
-        networks:
-            - kandalf
+  kafka:
+    image: wurstmeister/kafka:latest
+    restart: always
+    depends_on:
+      - zookeeper
+    ports:
+      - "9092"
```

Comment:
Is this file for local purpose? If yes then we probably need to bind local ports as we are doing that in hfkafka:
https://github.com/hellofresh/hfkafka/blob/master/docker-compose.yml#L36

Reply:
ok